src/content/dataSubmissions/DataSubmission.tsx

```diff
@@ -233,7 +253,31 @@ const columns: Column<Batch>[] = [
       minWidth: "240px"
     }
   },
-  /* TODO: Error Count removed for MVP2-M2. Will be re-added in the future */
+  {
+    label: "Error Count",
```

Comment:
Should the column header be "Error Count" or "Errors"? CRDCDH-496 seems to say "Errors"